src/main/java/uk/gov/hmcts/cmc/claimstore/utils/Optionals.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.cmc.claimstore.utils;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)